### DISCUSSION TITLE
Does anyone know hot to start rendering the items at the bottom of the container?

### DISCUSSION THREAD

**Discussion by josedugu** [petyosi/react-virtuoso#976] [Q&A]
I am building a chat webapp and everything works fine except this detail, the items starts rendering at the top of the container, as is being know most of the chats apps starts rendering at bottom, any ideas how to do that?
Thanks :D

**Comment by 5mars** [comment]
Have you found an answer yet I'm having a similar problem.

**Reply by petyosi** [reply]
Take a look at https://virtuoso.dev/virtuoso-message-list/, it has all configurations specific to the chat interface.
